.github/workflows/merge-to-stage.js

```diff
@@ -236,7 +236,8 @@ const main = async (params) => {
     const stageToMainPR = await getStageToMainPR();
     console.log('has Stage to Main PR:', !!stageToMainPR);
     if (stageToMainPR) body = stageToMainPR.body;
-    existingPRCount = body.match(/https:\/\/github\.com\/adobecom\/milo\/pull\/\d+/g).length;
+    console.log('Current PR body:', body);
+    existingPRCount = body.match(/https:\/\/github\.com\/adobecom\/milo\/pull\/\d+/g)?.length || 0;
```

Comment:
Ah nvm, we get a null value. My omission! 
![image](https://github.com/user-attachments/assets/fcb0ac13-4941-4fc0-bfac-8bd5990b0248)